cefdatabrowse/cefdatabrowse_support.py

```diff
@@ -0,0 +1,521 @@
+#!/usr/bin/env python
```

Comment:
If cefdatabrowse_support.py is really just another duplication of the same code found in databrowse.wsgi and db_lib, perhaps we should merge this all together so we aren't maintaining three copies of basically the same code.  I would suggest that we modify databrowse.wsgi and cefdatabrowse.py to simply call or wrap the library as appropriate.  This might require some cleanup, but I think it would be quite good to not have three copies of essentially the same code to maintain.